### ISSUE TITLE
A11y: View & Download buttons are missing `aria-label`

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#484]
## Describe the task
When we removed the tooltips from the View & Download buttons on the Find Properties page, we removed the content inside of the buttons so now when screenreaders land on the buttons all it says is "button." 

Add `aria-label="View"` and `aria-label="Download"` to these buttons.

## Acceptance Criteria

- [ ] Turn on Voiceover or other screenreader of your choice
- [ ] Navigate to the Find Properties page & the view/download buttons.
- [ ] When you navigate to the buttons, you will hear "View button" / "Download button"

**Comment by AZBL** [comment]
I'd like to be assigned this please.

**Comment by AZBL** [comment]
Hi @bacitracin just wanted to make sure you saw this request. Thanks!

**Comment by bacitracin** [comment]
@AZBL Whoops sorry missed this. Let me know if you have any questions!

**Comment by AZBL** [comment]
Hi @bacitracin, a few questions in regards to this. Mostly I just want to make sure I'm targeting the right things because I'm not sure I am.

-The View Button isn't present on the staging site. 
-In SidePanelControlBar.tsx the Download ThemeButton does have aria-label=Download and the screen reader voiceover does say "Download Button". On the main site my screen reader said "View Button" for that as well.
-I did notice the filter button doesn't have an aria-label and it just says "Filter". 

Thanks - Alex

**Comment by bacitracin** [comment]
@AZBL Hey there! I'm checking the staging environment, and it looks like this ticket isn't relevant anymore. The Filter button doesn't need an `aria-label` because the visible text is available and read out by the screenreader. 

Sorry for the confusion, the code is moving fast these days
